src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeGeneration/AbstractCodeGenerationService_FindDeclaration.cs

```diff
@@ -77,13 +77,15 @@ public bool CanAddTo(SyntaxNode destination, Solution solution, CancellationToke
                 return false;
             }
 
+#if !CODE_STYLE
             // If we are avoiding generating into files marked as generated (but are still regular files)
             // then check accordingly. This is distinct from the prior check in that we as a fallback
             // will generate into these files is we have no alternative.
             if (checkGeneratedCode && document.IsGeneratedCode(cancellationToken))
```

Comment:
IsGeneratedCode is not available in CODE_STYLE layer.  We only have IsGeneratedCodeAsync instead.  But trying to use that virally exploded into too much becoming async.  So i just carved this out as not supported.

Reply:
This sounds reasonable as code fixers likely won't be dealing with generated code.